lib/convert_json_schema.rb

```diff
@@ -56,8 +56,15 @@ def get_fields(schema)
   end
 
   def convert_to_json_schema(props)
+    is_required = true
+
     # Remove required if default is set
-    props.delete("required") unless props["default"].nil?
```

Comment:
IMO, it would be easier to read if we weren't using both `unless` as well as `if` in this function